frontend/src/pages/AccountSend.vue

```diff
@@ -398,6 +409,22 @@ function useSendForm() {
     sendFormRef.value?.resetValidation();
   }
 
+  async function setHumanAmountMax() {
+    if (!token.value?.address)  throw new Error(vm.$i18n.tc('Send.select-a-token'))
+    if (NATIVE_TOKEN.value?.address === token.value?.address) {
+        const address = await signer.value?.getAddress()
+        if (!address || !provider.value) throw new Error(vm.$i18n.tc('Send.wallet-not-connected'))
+        if (!recipientId.value) throw new Error(vm.$i18n.tc('Send.enter-a-recipient'))
+        const {ethToSend} = await umbraUtils.getEthSweepGasInfo(address || "", recipientId.value, provider.value)
+        humanAmount.value = formatUnits(ethToSend.toString(), token.value.decimals)
```

Comment:
For both `humanAmount.value =` lines, let's use the `humanizeTokenAmount` helper instead. There's a lot of quirks to make the numbers look nice with varying decimal numbers/magnitudes so that method handles all of them

Reply:
Thanks, updated to use that helper. Although, I noticed a bug when the value was a 1,000. It seems like the comma is causing an error to pop up.

Reply:
Oh interesting! Do you have more info / steps to reproduce? Would be great to track in an issue 🙂 

(unresolved to discuss that bug)

Reply:
Here is a video of a reproduction
[format_bug.webm](https://user-images.githubusercontent.com/12705146/184269664-1053d1b6-0cea-4005-8fda-a545209113ba.webm)

Reply:
Ahhh sorry yes this makes total sense, we should not have commas in the input field. Had a brain fart apparently. I'll push a commit putting back what you originally had

Reply:
Pushed f31e279 which should put it back, lmk if that looks right to you